.github/workflows/self_test.yaml

```diff
@@ -584,7 +584,7 @@ jobs:
 
           ### ⛓️ Dependencies
           - Upgraded newrelic/infrastructure from 0.0.0 to 0.0.1 - [Changelog 🔗](https://github.com/newrelic/infrastructure-agent/releases/tag/0.0.1)
-          - Upgraded newrelic/nri-docker from 0.0.3 to 0.0.4 - [Changelog 🔗](https://github.com/newrelic/nri-docker/releases/tag/0.0.4)
+          - Upgraded newrelic/nri-docker from 0.0.3 to 1.8.26 - [Changelog 🔗](https://github.com/newrelic/nri-docker/releases/tag/v1.8.26)
```

Comment:
Since it bumps a major now, I think the corresponding version above will also need a new major (v2.0.0 instead of v1.3.0)

Reply:
Moreover the test points to the local contrib action that points to V1, therefore it is a circular dependency with the release